pwiz_tools/Skyline/TestRunnerLib/RunTests.cs

```diff
@@ -332,6 +338,34 @@ public bool Run(TestInfo test, int pass, int testNumber, string dmpDir, bool hea
                     Environment.SetEnvironmentVariable(@"SKYLINE_TESTER_PARALLEL_CLIENT_ID", ParallelClientId); // Accessed in pwiz_tools\Skyline\Util\Util.cs
                 }
 
+                // Set the temp file path to something peculiar - helps guarantee support for
```

Comment:
Put this in a function instead of making a BIG function bigger.

Reply:
If this gets moved to a function it needs to be a functor with a Dispose call that resets the TMP variable (as is done later in this function). But having that environment-restoring behavior is probably a good idea anyway if the code throws between the time TMP is redirected and when it is restored.

Reply:
Or the function needs to return tmpTestDir?